### DISCUSSION TITLE
Longhorn backup and snapshot are increasing my pv size actual size used by pv is lesser

### DISCUSSION THREAD

**Discussion by omgupta240909** [longhorn/longhorn#8367] [General]
Hello @Team,

The size of the pv inside the pod is 8.8Gb
<img width="758" alt="image" src="https://github.com/longhorn/longhorn/assets/153708380/197759f6-166d-4c0c-ace2-2b2a98d5a13f">

But in the longhorn Ui it is showing me 66Gi and when i go to where my volume is sceduled there also it is showing me the 66Gi

<img width="382" alt="image" src="https://github.com/longhorn/longhorn/assets/153708380/abd8271c-1fab-4ed1-a4c0-1e8ca61b5ebf">

I have configred a daily backup of the volume and the system backup at once in a week , for the backup we have nfs4 configured , so why it is happening ?
please help.

Thanks,
om

**Comment by mantissahz** [comment]
Hi @omgupta240909,

Because the replica is a sparse file as mentioned in this section https://longhorn.io/docs/1.6.1/concepts/#23-replicas

You can do `Trim Filesystem` manually on the Volume page of the Longhorn UI.
Or you could create a recurring job to do `Filesystem Trim` daily on the Recurring Job page of the Longhorn UI.

**Reply by omgupta240909** [reply]
Hi @mantissahz  
Thanks for your reply, in my pv why it is storing backup and system backup.
![image](https://github.com/longhorn/longhorn/assets/153708380/e5c43331-a54f-4c49-a4ec-f78b94544c6f)

When  i create a job for trim file system will it affect my data ?

**Comment by mantissahz** [comment]
>Thanks for your reply, in my pv why it is storing backup and system backup.

That's snapshots taken for backups and system backups.

>When i create a job for trim file system will it affect my data ?

The operation is equal to the command `fstrim` and usually it should not affect your data.

**Reply by omgupta240909** [reply]
Hello @mantissahz  

I  have trim one of my volume but its size didn't changed.
how do i delete the older snapshots is there any mechanism to delete it , and if i delete will it affect my data ? 
![image](https://github.com/longhorn/longhorn/assets/153708380/5e0f1a84-29c2-4ce0-bc57-83963a55f428)

**Reply by mantissahz** [reply]
>I have trim one of my volume but its size didn't changed.

How did you trim your volume by Longhorn UI? Can you provide the information by the command `du -h` on nodes?

>how do i delete the older snapshots is there any mechanism to delete it , and if i delete will it affect my data ?

You can set the `retain` number when you create a recurring job.
Or you can delete snapshots on the Volume Details page of Longhorn UI.

For System backup, you should be able to delete them on the System Backup Page.
